migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql

```diff
@@ -0,0 +1,206 @@
+-- IDEMPIERE-5932 - better menu and window names
+SELECT register_migration_script('202312080607_IDEMPIERE-5932.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Dec 8, 2023, 6:07:00 AM CST
+UPDATE AD_Menu SET Name='Quote-to-Invoice (Sales) ',Updated=TO_TIMESTAMP('2023-12-08 06:07:00','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=166
+;
+
+-- Dec 8, 2023, 6:07:22 AM CST
+UPDATE AD_Menu SET Name='Requisition-to-Invoice (Purchasing)',Updated=TO_TIMESTAMP('2023-12-08 06:07:22','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=203
+;
+
+-- Dec 8, 2023, 6:08:15 AM CST
+UPDATE AD_Menu SET Name='Open Items (Financial movements and Aging)',Updated=TO_TIMESTAMP('2023-12-08 06:08:15','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=236
+;
+
+-- Dec 8, 2023, 6:08:21 AM CST
+UPDATE AD_Menu SET Name='Open Items (Aging)',Updated=TO_TIMESTAMP('2023-12-08 06:08:21','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=243
```

Comment:
The report title is **Open Item (Invoice) List**, perhaps the two should be in sync ?